backend/apps/driving/models.py

```diff
@@ -0,0 +1,20 @@
+from django.core.validators import MaxLengthValidator
+from django.db import models
+
+
+class Chapter(models.Model):
+    text = models.TextField(validators=[MaxLengthValidator(2000)])
```

Comment:
Probably (but I definitely do not insist) there is a sense to declare `id` field explicitly, like this:

```python
id = models.BigIntegerField(primary_key=True)
```

In this case it will not be auto-increment, but it should be OK, as we'll set it directly, when reading data from JSON.